en/docs/install-and-setup/deploying-wso2-api-manager/production-deployment-guidelines.md

```diff
@@ -54,13 +60,12 @@ Prior to installing any WSO2 Carbon-based product, it is necessary to have the a
 </tbody>
 </table>
 
-#### 
-Environment compatibility
+### Environment compatibility
 
 -   By default, WSO2 products are installed with **OpenJDK** , which allows you to run the product as soon as it is installed.
 
-        !!! tip
-    To use a different JDK, point the **`JAVA_HOME           `** environment variable to the new JDK. Make sure your JDK version is [compatible with the WSO2 product](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems+and+JDKs) .
+    !!! tip
+        To use a different JDK, point the **`JAVA_HOME           `** environment variable to the new JDK. Make sure your JDK version is [compatible with the WSO2 product](https://docs.wso2.com/display/compatibility/Tested+Operating+Systems+and+JDKs) .
```

Comment:
Shall we point this to {base_path}/install-and-setup/product-compatibility/#product-compatibility